### ISSUE TITLE
many "unlabeled" buttons with talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [nogringo/nostr-mail-client#23]
When using the Talkback screen reader on Android, there are quite a few buttons in this application that read as "unlabeled". The first one you encounter is in the setup flow screens, the next arrow button that takes you to the next page, for example page 205, page 305, page 405, page 405, etc. is unlabeled.

Once you enter the main screen, the nav menu button, the search button, the settings button, the account button, and the compose button read as "unlabeled"

<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/c6ff74fa-db51-4daf-88be-93bdbc5c218c" />

In the compose screen, the attach button to the right of the subject field reads as unlabeled, and so do the bold, italic, underline, etc. buttons in the row beneath the subject, all except for the last one, which seems like an attached link button.

<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/4375bf00-7159-460b-83e5-49bae9a5f29d" />

The color square and plus icon in the next square beneath "Dynamic theme
Generate colors from background image" also reads as unlabeled.

<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/62d8455c-139a-4fec-8870-ce26b9bb532d" />

**Comment by nogringo** [comment]
It should be fixed in the version 0.12.0. Can you verify it ?

**Comment by shortwavesurfer2009** [comment]
Perfect! Closing this as completed.
